### ISSUE TITLE
wrong linebreak

### ISSUE THREAD

**Issue by OskarStark** [FriendsOfPHP/PHP-CS-Fixer#1909]
```patch
@@ -49,7 +49,8 @@ class LivestreamControllerTest extends WebTestCase
         $this->assertSame('application/json', $response->headers->get('Content-Type'));

         $content = json_decode($response->getContent());
-        $embeddedRelations = (array) $content->_embedded->items{0}->_embedded;
+        $embeddedRelations = (array) $content->_embedded->items{0}
+        ->_embedded;

         /*
          * Assert that channel is embedded
```

Using:
`PHP CS Fixer version 1.11.1 by Fabien Potencier`

**Comment by OskarStark** [comment]
The [PHP doc](http://php.net/manual/en/language.types.array.php#99015) allows both:

> Both square brackets and curly braces can be used interchangeably for accessing array elements (e.g. $array[42] and $array{42} will both do the same thing in the example above).

**Comment by keradus** [comment]
Please, could you confirm the problem using v2@dev ?

**Comment by SpacePossum** [comment]
Confirmed on 1.12 and master as well, fixer doing this is `braces`.
Input used for test;
```php
<?php

/*
 * This file is part of PHP CS Fixer.
 *
 * (c) Fabien Potencier <fabien@symfony.com>
 *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
 *
 * This source file is subject to the MIT license that is bundled
 * with this source code in the file LICENSE.
 */

class A
{
    private function test($content)
    {
        $embeddedRelations = (array) $content->_embedded->items{0}->_embedded;
    }
}

```

**Comment by OskarStark** [comment]
Using:
`PHP CS Fixer version 2.0-DEV by Fabien Potencier and Dariusz Ruminski`

@SpacePossum was faster 😄 

Yes i can confirm this bug

**Comment by keradus** [comment]
1. need a new transformer (then 2.x line) for `{` used for indexing purposes
2. need a nef fixer to convert `{` to `[`

**Comment by keradus** [comment]
@OskarStark could you temporary switch to #1955 and check out bug you detected ?

**Comment by keradus** [comment]
New fixer to normalize index parenthesis: #1958

**Comment by keradus** [comment]
Fixed by #1955. Sadly, fix was a BC breaker, so it's fixed only in 2.x version.
If anyone want to port the fix to 1.11 feel free to do so

**Comment by OskarStark** [comment]
> @OskarStark could you temporary switch to #1955 and check out bug you detected ?

Sorry, i didn't have time the last 2 days, but i will try the 2.x version...
